Dockerfile

```diff
@@ -11,20 +11,24 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.2.1" \
 
 WORKDIR /opt/atomicapp
 
-# add all of Atomic App's files to the container image
-ADD atomicapp/ /opt/atomicapp/atomicapp/
-ADD setup.py requirements.txt /opt/atomicapp/
+# Add the requirements file into the container
+ADD requirements.txt /opt/atomicapp/
 
-# lets install pip, and gcc for the native extensions
-# and remove all after use
-RUN yum -y install epel-release && \
-    yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc && \
-    python setup.py install && \
-    yum remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
+# Install needed requirements
+RUN yum install -y epel-release \
+    yum install -y --setopt=tsflags=nodocs docker && \
+    yum install -y --setopt=tsflags=nodocs $(sed s/^/python-/ requirements.txt) && \
```

Comment:
we are no longer installing gcc etc.. any more. only docker and the python dependencies via rpm. We are not using pip to get deps and then compiling them, so no need for gcc/glibc-devel.